app/assets/javascripts/channels/admin/repo_activity.js

```diff
@@ -0,0 +1,13 @@
+const RepoActivity = () => {
+  window.App = window.App || {}
+  App.cable = ActionCable.createConsumer('/admin/cable')
+
+  App.cable.subscriptions.create(
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
'App' is not defined. (no-undef)